src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/ApplicationService.kt

```diff
@@ -418,6 +423,7 @@ class ApplicationService(
             offenceId,
             riskRatings,
             offenderDetails,
+            prisonName,
```

Comment:
Just to draw your to CAS2's similar work: https://github.com/ministryofjustice/hmpps-approved-premises-api/pull/1284 We opted to store the prison _code_ (AgencyId) rather than the prison _name_ as apparently prison names change often enough that it would be an issue in reporting.